ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -120,7 +120,7 @@ message AllocatePartyRequest {
   string display_name = 2;
 
   // Optional
-  repeated Property properties = 3;
+  ObjectMeta object_meta = 3;
```

Comment:
```suggestion
  ObjectMeta metadata = 3;
```
Following k8s naming; and also as `object` does not mean a lot in this context.

Reply:
Ok